packages/lib-helpers/react/package.json

```diff
@@ -11,6 +11,6 @@
   "devDependencies": {
     "@types/react": "^18.2.56",
     "@types/react-dom": "^18.2.19",
-    "typescript": "^5.3.3"
+    "typescript": "5.3.3"
```

Comment:
Hey @eunjae-lee 🙌 
Should the same change also be applied to the other packages using typescript? 🤔 

I mean, there are other projects using TS but not enforcing it to be the exact version as here.

Reply:
Thanks! Applied https://github.com/storyblok/field-plugin/pull/411/commits/5dc97986f28407a5f01b04e3a7ba642dd8abf7c9